doc/backend_vars.asciidoc

```diff
@@ -8,7 +8,7 @@ Supported variables per backend
 Variable;Values allowed;Default value;Explanation
 INCLUDE_MODULES;string;;comma separated names or fullnames of test modules to be included while excluding all that do not match, e.g. "boot,mod1"
 EXCLUDE_MODULES;string;;comma separated names or fullnames of test modules to exclude. Can be combined with INCLUDE_MODULES and has precedence, e.g. to additionally exclude modules based on an include-list
-SCHEDULE;string;;comma separated list of relative paths to test modules within CASEDIR without the implicit file extension '.pm' to be scheduled instead of evaluating a schedule from the test distributions main.pm file, e.g. "boot,console/mod1"
+SCHEDULE;string;;comma separated list of relative paths to test modules within *CASEDIR* to be scheduled. _e.g. "SCHEDULE=boot/boot_to_desktop,console/mod1"_.
```

Comment:
That's not right. Why did you delete the part about the extension? Despite what os-autoinst-distri-opensuse does, isotovideo in this case does the same. Using more asciidoc formating is nice but then I would recommend to do that in the whole file for more variables. What you should focus on here is the explanation itself. That has not improved in the latest version of your change